fdbserver/Status.actor.cpp

```diff
@@ -2419,6 +2432,9 @@ ACTOR Future<StatusReply> clusterGetStatus(
 				statusObj["old_logs"] = oldTlogFetcher(&oldLogFaultTolerance, db, address_workers);
 			}
 
+			// Used as a signal that storage servers may not be able to catch up certain log generations
+			statusObj["possibly_losing_old_logs_data"] = oldLogFaultTolerance < 0;
```

Comment:
Same as above, only report if `recoveryState < FULLY_RECOVERED`?

Reply:
I think we are fine here because If recoverState is >= STORAGE_RECOVERED, we must have an empty list of old tlogs and in this case we won't report anything in cli. 

I also add reporting current log generation in the new change. Even when recoveryState == FULLY_RECOVERED, we still want to report missing tlogs in the current generation. Therefore we may don't want to add a condition recoveryState < FULLY_RECOVERED here.